Libplanet/Net/NetMQTransport.cs

```diff
@@ -410,20 +468,41 @@ public async Task RunAsync(CancellationToken cancellationToken)
                 cancellationToken);
         }
 
+        /// <summary>
+        /// Visualizes Kademlia-based routing table status.
+        /// </summary>
+        /// <remarks>Will be outdated.</remarks>
+        /// <returns><see cref="string"/> representation of the routing table.</returns>
         public string Trace() => Protocol is null ? string.Empty : Protocol.Trace();
 
+        /// <inheritdoc />
         public void Dispose()
         {
             _runtimeCancellationTokenSource.Cancel();
             _turnCancellationTokenSource.Cancel();
             _runtimeProcessor.Wait();
         }
 
+        /// <summary>
+        /// Task complete when the transport is running.
+        /// </summary>
+        /// <seealso cref="Swarm{T}.WaitForRunningAsync()"/>
+        /// <returns>An awaitable task without value.</returns>
         public Task WaitForRunningAsync() => _runningEvent.Task;
```

Comment:
It will be better to reference `Swarm<T>.WaitForrunningAsync()`.

It says:

```csharp
        /// <summary>
        /// Waits until this <see cref="Swarm{T}"/> instance gets started to run.
        /// </summary>
        /// <returns>A <see cref="Task"/> completed when <see cref="NetMQTransport.Running"/>
        /// property becomes <c>true</c>.</returns>
        public Task WaitForRunningAsync() => (Transport as NetMQTransport)?.WaitForRunningAsync();
```